### PR TITLE
[Communication] - Phone Numbers - Test Proxy migration

### PR DESCRIPTION
**Pull request by danielortega-msft**
# Description

Please add an informative description that covers that changes made by the pull request and link all relevant issues.

If an SDK is being regenerated based on a new swagger spec, a link to the pull request containing these swagger spec changes has been included above.

# All SDK Contribution checklist:
- [ ] **The pull request does not introduce [breaking changes]**
- [ ] **CHANGELOG is updated for new features, bug fixes or other significant changes.**
- [ ] **I have read the [contribution guidelines](https://github.com/Azure/azure-sdk-for-python/blob/main/CONTRIBUTING.md).**

## General Guidelines and Best Practices
- [ ] Title of the pull request is clear and informative.
- [ ] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For more information on cleaning up the commits in your PR, [see this page](https://github.com/Azure/azure-powershell/blob/master/documentation/development-docs/cleaning-up-commits.md).

### [Testing Guidelines](https://github.com/Azure/azure-sdk-for-python/blob/main/CONTRIBUTING.md##building-and-testing)
- [ ] Pull request includes test coverage for the included changes.

### CODE REVIEW

**Review by jiriburant — APPROVED** [review]

**Comment by danielortega-msft** [comment]
Related issue: #26512

File: sdk/communication/azure-communication-phonenumbers/tests.yml [hunk 1]
```diff
@@ -21,6 +21,9 @@ stages:
           MatrixReplace:
             - COMMUNICATION_SKIP_CAPABILITIES_LIVE_TEST=false/true
       Clouds: Public,Int
+      EnvVars:
+        AZURE_SKIP_LIVE_RECORDING: 'True'
+        AZURE_TEST_RUN_LIVE: 'true'
```
Comment (sofiar-msft) [inline sdk/communication/azure-communication-phonenumbers/tests.yml:26]: Both should be "True" with capitalized T

**Review by mccoyp — APPROVED** [review]

File: sdk/communication/azure-communication-phonenumbers/tests.yml (hunk 1 above)
Comment (danielortega-msft) [reply sdk/communication/azure-communication-phonenumbers/tests.yml:26]: This is actually taken straight from the migration guide: https://github.com/Azure/azure-sdk-for-python/blob/main/doc/dev/test_proxy_migration_guide.md#live-test-pipelines. I can make the change, but the thing is that I am not sure if this is used by Python or if the value is used by the pipeline script itself.

Comment (mccoyp) [reply sdk/communication/azure-communication-phonenumbers/tests.yml:26]: You can align on either for consistency, but the values should actually be case-insensitive when used by pipelines

**Pull request merged**